From ef3a5b4aea219a877d17c30f9c9cb91eca122dd3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E2=80=98mldn=E2=80=99?= <‘mldnqa@163.com’>
Date: Sun, 14 Mar 2021 23:58:33 +0800
Subject: [PATCH] change Admin.java

---
 Admin.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Admin.java b/Admin.java
index 18b3fb6..1fe52e8 100644
--- a/Admin.java
+++ b/Admin.java
@@ -2,5 +2,6 @@ import java.util.List;
 public class Admin{
 	public static void main(String args[]){
 		private List<Emp> emps;
+		private String name;
 	}
 }
\ No newline at end of file
-- 
2.30.0.windows.1

